files/initramfs-tools/union-mount.j2

```diff
@@ -38,6 +39,9 @@ for x in $(cat /proc/cmdline); do
         logs_inram=on)
             logs_inram=true
             ;;
+        mount_pmem=on)
```

Comment:
@vdahiya12 adding `memmap=*` is sufficient.

Reply:
fixed